index.d.ts

```diff
@@ -0,0 +1,460 @@
+	*/
+	inputMatches: (list: (string | ScriptFilterItem)[], target?: string | ((item: string | ScriptFilterItem, input: string) => boolean)) => (string | ScriptFilterItem)[];
+
+	/**
+	Log value to the Alfred workflow debugger.
+	@param text
+	*/
+	log: (text: string) => void;
+
+	/**
+	Display an error or error message in Alfred.
+	You don't need to .catch() top-level promises.
+	Alfy handles that for you.
+	@param error
+	*/
+	error: (error: Error | string) => void;
+
+	/**
+	Returns a Promise that returns the body of the response.
+	@param url
+	@param options
+	@returns Body of the response.
+
+	@example
+	```
+	await alfy.fetch('https://api.foo.com', {
+		transform: body => {
+			body.foo = 'bar';
+			return body;
+		}
+	})
+	```
+	*/
+	fetch: (url: GotUrl, options?: FetchOptions) => Promise<unknown>;
+
+	/**
+	@example
+	```
+	{
+		name: 'Emoj',
+		version: '0.2.5',
+		uid: 'user.workflow.B0AC54EC-601C-479A-9428-01F9FD732959',
+		bundleId: 'com.sindresorhus.emoj'
+	}
+	```
+	*/
+	meta: {
+		name: string;
+		version: string;
+		bundleId: string;
+		type: string;
+	};
+
+	/**
+	Input from Alfred. What the user wrote in the input box.
+	*/
+	input: string;
+
+	/**
+	Persist config data.
+	Exports a conf instance with the correct config path set.
+
+	@example
+	```
+	alfy.config.set('unicorn', '🦄');
+
+	alfy.config.get('unicorn');
+	//=> '🦄'
+	```
+	*/
+	config: Conf<unknown>;
+
+	/**
+	Persist cache data.
+	Exports a modified conf instance with the correct cache path set.
+
+	@example
+	```
+	alfy.cache.set('unicorn', '🦄');
+
+	alfy.cache.get('unicorn');
+	//=> '🦄'
+	```
+	*/
+	cache: CacheConf<unknown>;
+
+	/**
+	Get various default system icons.
+	The most useful ones are included as keys. The rest you can get with icon.get().
+	Go to /System/Library/CoreServices/CoreTypes.bundle/Contents/Resources in Finder to see them all.
+
+	@example
+	```
+	console.log(alfy.icon.error);
+	//=> '/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertStopIcon.icns'
+
+	console.log(alfy.icon.get('Clock'));
+	//=> '/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/Clock.icns'
+	```
+	*/
+	icon: {
+		get: (icon: string) => string;
+		info: string;
+		warning: string;
+		error: string;
+		alert: string;
+		like: string;
+		delete: string;
+	};
+
+	/**
+	Alfred metadata.
+	*/
+	alfred: {
+		version: string;
+		theme: string;
+		themeBackground: string;
+		themeSelectionBackground: string;
+		themeSubtext: string;
+		data: string;
+		cache: string;
+		preferences: string;
+		preferencesLocalHash: string;
+	};
+
+	/**
+	Whether the user currently has the workflow debugger open.
+	*/
+	debug: boolean
+
+	/**
+	Exports a Map with the user workflow configuration.
+	A workflow configuration allows your users to provide configuration information for the workflow. 
+	For instance, if you are developing a GitHub workflow, you could let your users provide their own API tokens.
+	See alfred-config for more details.
```

Comment:
Should be linkified like the readme.

Make sure that all the text you copied from the readme retains its style, links, and inline code blocks.